README.md

```diff
@@ -41,6 +41,10 @@ To run all tests use:
 cd BladeMonRT
 go test ./...
 
+To run the end-to-end test:
+rename test_end_to_end.go to test_end_to_end_test.go
+go test -run TestEndToEnd
```

Comment:
Explain what this does:
- Runs BRT until a keyboard interrupt
- you will have to manually raise ETW events in a separate Terminal